aws-sdk-core/lib/aws-sdk-core/ini_parser.rb

```diff
@@ -0,0 +1,77 @@
+module Aws
+  class IniParser
+
```

Comment:
Done. Also removed an unused method from the class (development artifact I neglected to remove).